### DISCUSSION TITLE
Doesn't seem a way to simply email the makers . Sooo

### DISCUSSION THREAD

**Discussion by Tech-Idiot** [d4rken-org/sdmaid-se#1448] [Q&A] [answered]
Where is says SD card does ticking mean it will scan the card or will unticking scan the card?

![Screenshot_20241102-010108](https://github.com/user-attachments/assets/87d80936-89cd-4168-bd0f-de8c462aa1c6)

I'm sure this will enrage all the GitHub users 

Don't really know what GitHub is.

Just couldn't find a way to find the answer.

Anyway I'm sure you are all amazing!!

Keep hubing that git 😊👍💪 for all of us who haven't a clue.

**Comment by d4rken** [comment, marked as answer]
Ticking it will include that data area.
